twig/BS3/macros/forms.twig

```diff
@@ -38,23 +38,26 @@
                 {% endif %}
 
                 <div class="form-group {{ field.getErrors()|length>0?'has-error has-feedback'}} {{ field.getNoticeText()|length>0?'has-warning has-feedback' }}">
-                    {% if field.label is not null  %}
-                        <label {{ (field.isSuppressLabel() ? 'class="sr-only"') }} for="{{ field.id }}">
-                            {{ field.label|raw }}
-                            {% if field.reqLevel > 1 %}
-                                {{ myself.requiredIndicator(field) }}
-                            {% endif %}
+                    {% set labelIdSwitch = {0:'TYPE_TOGGLE_TRUE_FALSE', 1:'TYPE_TOGGLE_YES_NO', 2:'TYPE_STATIC'} %}
```

Comment:
Sorry, this is a straight up no.  We're not coding in static arrays like this.  Since this is a library, anyone can add their own fields.  See src/FormFields in our own project.

Reply:
Should I add this to the definitions (like getUseStandardLabel() or something)?  The big problem that we have is labeling divs- for WCAG stuff, labels are only for form elements like input, select, etc.  For things like toggles that require a field set, the standard labeling in forms.twig needs that little tweak.  I'm open to the best way to do this!

Reply:
Would it be better to use fieldsets with legends instead of divs with labels everywhere?

Reply:
It's best to use them for multi-input type things (like radios and check boxes) or sets of data (address).  According to WCAG standards, the fieldset and legend should be used to describe a set of inputs that relate to each other in some way.  Each input in a fieldset should have its own label.  The legend should describe the group of inputs.  In the ACEMAPP side of things, I also added a fieldset around the address stuff- it's all address stuff, and each item is individually labeled (as city, state, street etc) even though the labels are sr-only.

I think adding a fieldset around each input would make things worse- they would hear the fieldset legend, then the label, which would probably be the same thing :\

Reply:
Hence the `instead of` on that last bit.  I'm suggesting removing the label and running with fieldset/legend only instead of div/label.

When it comes to reading, is there any difference between reading the fieldset legend vs reading the div label?

Reply:
Discussed in person.   Mark this one resolved when complete.